conda/recipes/ucx-py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = 'ucx-py' %}
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version = environ.get('CONDA_PY', '38') %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.2').split('.')[:2]) %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
the `ucx-py` string is already lowercase, so having `|lower` here seems a bit extraneous. can it be removed? alternatively, can we just inline the `ucx-py` package name here instead of using a variable?

Reply:
Yeah this just came from [here]( https://github.com/rapidsai/cusignal/blob/600eb63d48ecb21ef4a374bbb91a0f6fa34b4828/conda/recipes/cusignal/meta.yaml#L8 ). Though agree we can simplify

Reply:
Inlined